CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -540,7 +540,16 @@ macro(slicerMacroBuildApplication)
       # Define list of extra 'application to launch' to associate with the launcher
       # within the build tree
       set(extraApplicationToLaunchListForBuildTree)
-      if(EXISTS "${QT_DESIGNER_EXECUTABLE}")
+
+      if(${Slicer_REQUIRED_QT_VERSION} VERSION_GREATER_EQUAL 5 AND NOT QT_DESIGNER_EXECUTABLE)
+        # Since Qt only provides a CMake module to find the designer library, we work
+        # around this limitation by deducting the path of the designer executable.
+        if(EXISTS ${QT_BINARY_DIR}/designer${CMAKE_EXECUTABLE_SUFFIX})
+          set(QT_DESIGNER_EXECUTABLE ${QT_BINARY_DIR}/designer${CMAKE_EXECUTABLE_SUFFIX})
```

Comment:
On Mac Qt Designer is in Designer.app, so this won't work.

Reply:
Really an upstream variable is the way to go. For now could someone submit a mac specific patch?

Reply:
You can use `find_program(QT_DESIGNER_EXECUTABLE designer Designer "${QT_BINARY_DIR}")`.

Reply:
@adamrankin  The first patch was integrated (by mistake) without referencing you as author. This will be fixed and was documented [here](https://www.slicer.org/wiki/Documentation/Labs/TransitionToGit#Authorship).

I am now pushing an updated patch to address macOS.